### DISCUSSION TITLE
Adding line breaks in text so they are not ignored when pasted into Outlook Web email, for example

### DISCUSSION THREAD

**Discussion by PeteBeet** [guilryder/clavier-plus#45] [Q&A]
I've tried several solutions but can't get Write Text to reliably paste line breaks into Outlook Web emails, when composing/replying to a message. "Embedded" line breaks in the text clearly work when the text is pasted into legacy Outlook, Word etc. but it's a problem with Outlook Web. 

BTW if I use Clavier+ to paste the text, for example, into Notepad and _then_ into an Outlook Web message, the line breaks format properly.

**Comment by guilryder** [comment]
Please refer to the FAQ for this type of question.

**Comment by PeteBeet** [comment]
Many thanks for your reply.

I have read the FAQ and can’t figure out how to accomplish what I’ve requested.

Paul

Paul Blank
***@***.***
917 838-1124

**Reply by guilryder** [reply]
FAQ entries 2, 3, and 16 are relevant.

**Comment by PeteBeet** [comment]
Thank you again for your reply.

[][Enter][][][Enter][]

Seems to work. First time I tried, before emailing you, it didn’t work properly. Probably a typo on my end.

Cheers,

Paul
